module.go

```diff
@@ -91,14 +93,34 @@ func (m *Module) Run(ctx context.Context) error {
 		return xerrors.Errorf("subscribing to topic: %w", err)
 	}
 
+	h, err := newHost(m.id, m.Manifest, func(ctx context.Context, b []byte) error {
+		return m.topic.Publish(ctx, b)
+	}, m.sigs, m.verif, m.log)
+	if err != nil {
+		return xerrors.Errorf("creating gpbft host: %w", err)
+	}
+
+	go func() {
+		err := h.Run(ctx)
+		m.log.Errorf("running host: %+v", err)
+	}()
 	for {
 		msg, err := sub.Next(ctx)
 		if err != nil {
 			m.log.Errorf("pubsub subscription.Next() returned an error: %+v", err)
 			break
```

Comment:
It seems slightly odd to swallow the original error here, break out of the loop and potentially return some other error that may be unrelated to the original error that triggered the whole thing.

Reply:
This error won't get captured either way, I can wrap it but it will end up just logged